src/modules/authentication/api/authentication.controller.ts

```diff
@@ -118,4 +122,21 @@ export class AuthenticationController {
         );
         return new UserinfoResponse(permissions, rolleFieldsResponse);
     }
+
+    @Get('resetPassword')
```

Comment:
Better call the endpoint **reset-password**, it's better readable and prevents issues with case sensitivity or insensitivity.